features/step_definitions/wallets-steps.js

```diff
@@ -92,6 +100,67 @@ export default function () {
     return this.client.click('.WalletCreateDialog .dialog_button');
   });
 
+  this.When(/^I see the create wallet privacy dialog$/, function () {
+    return this.client.waitForVisible('.WalletBackupPrivacyWarningDialog');
+  });
+
+  this.When(/^I click on "Please make sure nobody looks your screen" checkbox$/, function () {
+    return this.waitAndClick('.WalletBackupPrivacyWarningDialog .CheckboxWithLongLabel_checkbox');
+  });
+
+  this.When(/^I submit the create wallet privacy dialog$/, function () {
+    return this.waitAndClick('.WalletBackupPrivacyWarningDialog .dialog_button');
+  });
+
+  this.When(/^I see the create wallet recovery phrase display dialog$/, function () {
+    return this.client.waitForVisible('.WalletRecoveryPhraseDisplayDialog');
+  });
+
+  this.When(/^I note down the recovery phrase$/, async function () {
+    const recoveryPhrase = await this.client.getText('.WalletRecoveryPhraseMnemonic_component');
+    this.recoveryPhrase = recoveryPhrase.split(' ');
+  });
+
+  this.When(/^I submit the create wallet recovery phrase display dialog$/, function () {
+    return this.waitAndClick('.WalletRecoveryPhraseDisplayDialog .dialog_button');
+  });
+
+  this.When(/^I see the create wallet recovery phrase entry dialog$/, function () {
+    return this.client.waitForVisible('.WalletRecoveryPhraseEntryDialog');
+  });
+
+  this.When(/^I click on recovery phrase mnemonics in correct order$/, async function () {
+    for (let i = 0; i < this.recoveryPhrase.length; i++) {
+      const recoveryPhraseMnemonic = this.recoveryPhrase[i];
+      await this.waitAndClick(`//button[contains(text(), "${recoveryPhraseMnemonic}") and @class="MnemonicWord_component MnemonicWord_active"]`);
+    }
+  });
+
+  this.When(/^I click on the "Accept terms" checkboxes$/, async function () {
+    const termsCheckboxes = await this.client.elements('.CheckboxWithLongLabel_checkbox');
+    for (let i = 0; i < termsCheckboxes.value.length; i++) {
+      const termsCheckbox = termsCheckboxes.value[i].ELEMENT;
+      await this.client.elementIdClick(termsCheckbox);
+    }
+  });
+
+  this.When(/^I submit the create wallet recovery phrase entry dialog$/, function () {
+    return this.waitAndClick('.WalletRecoveryPhraseEntryDialog .dialog_button');
+  });
+
+  this.Then(/^I should not see the create wallet recovery phrase entry dialog anymore$/, async function () {
+    await this.client.waitForVisible('.WalletRecoveryPhraseEntryDialog', null, true);
+    const result = await this.client.executeAsync(function(done) {
```

Comment:
I would actually split this part below out into a separate step definition as it does not quite fit to the title "should not see the create wallet dialog" 😉